documentation/modules/canceling-migration-cli.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_content-type: PROCEDURE
 [id="canceling-migration-cli_{context}"]
-= Canceling a migration
+= Canceling a migration from the CLI
 
-You can cancel an entire migration or individual virtual machines (VMs) while a migration is in progress from the command line interface (CLI).
+You can cancel either an entire migration, or the migration of specific virtual machines (VMs), while a migration is in progress from the command line interface (CLI).
```

Comment:
```suggestion
You can use the command-line interface (CLI) to cancel either an entire migration or the migration of specific virtual machines (VMs) while a migration is in progress.
```
Just a suggestion. But note that command-line should be written with a hyphen per IBM SG :-)

Reply:
You're right about "command-line interface", but IBM also has "On the command line, enter..." It prefers "command line" unless you need to distinguish the CLI from the UI, as I do here. Just to add a little something...apologies if you knew this already.